pyfar/dsp/levels.py

```diff
@@ -0,0 +1,63 @@
+"""
+Functions used to calculate the levels of signals.
+"""
+
+import numpy as np
+import scipy.signal as sps
+
+
+def time_weighted_levels(signal, weighting: str,
+                         reference=1) -> np.ndarray:
+    """
+    Calculates sound pressure levels with exponential time weighting
+    according to IEC / DIN EN 61672-1.
+
+    Paramters:
+    ---------
+
+    signal: Signal
+        The signal object to calculate the levels for
+
+    type: str
+        The time weighting type. Options are "F" (="fast") and "S" (="slow"),
```

Comment:
We allways render actual inputs as code and for being verbose I would suggest to expect full names
```suggestion
        The time weighting type. Options are ``'fast``' and ``'slow``',
```

Reply:
The norm defines the two types as "F" and "S" and adds a side note that they stand for fast and slow. So the official names are "F" and "S", though it was my idea to allow the full names too for ease of use.